Game/Assets/Coliseo/Scripts/AdvVRTracker.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
+using System.IO;
+using UnityEngine;
+
+public class AdvVRTracker : MonoBehaviour {
+
+    UnityEngine.VR.VRNode VRHead = UnityEngine.VR.VRNode.Head;
+    TextWriterTraceListener listener;
+    Transform cameraTransform;
+    Transform cameraRigTransform;
+    Transform headTransform;
+    Transform spineTransform;
+    Transform neckTransform;
+    Animator anim;
+
+    Vector3 rotationNeeded;
+    Vector3 positionNeeded;
+
+    Vector3 initHeadRot;
+    
+    bool logging = false;
+
+    // These are all specific to the scientist model
+    static float distHipsToSpine = 0.08003899f;
+    static float distSpineToSpine1 = 0.18539f;
+    static float distSpine1ToNeck = 0.264977f;
+    static float distNeckToHead = 0.115351f;
+
+    static float distSpineToNeck = distSpineToSpine1 + distSpine1ToNeck;
+    static float distHipsToNeck = distHipsToSpine + distSpineToNeck;
+    static float distHipsToHead = distHipsToNeck + distNeckToHead;
+
+    float riftZ;
+    float riftY;
+
+    float h;
+    float k;
+    float p;
+    float q;
+
+    float rotSpine;
+    float rotNeck;
+
+
+    // This script should actually be moved to the main thing, honestly. And various
+    // things should be edited to reflect that. Use anim.GetBoneTransform instead
+    void Start()
+    {
+        anim = GetComponent<Animator>();
+        // set up listener
+        string filename = @"./output.txt";
+        FileStream traceLog = new FileStream(filename, FileMode.OpenOrCreate);
+        if (logging)
+        {
+            listener = new TextWriterTraceListener(traceLog);
+        }
+        
+        headTransform = anim.GetBoneTransform(HumanBodyBones.Head);
+        neckTransform = anim.GetBoneTransform(HumanBodyBones.Neck);
+        spineTransform = anim.GetBoneTransform(HumanBodyBones.Spine);
+        cameraRigTransform = headTransform.Find("CameraRig");
+        cameraTransform = cameraRigTransform.Find("Camera");
+
+        initHeadRot = headTransform.localRotation.eulerAngles;
+        
+    }
+
+    // Update is called once per frame
+    void LateUpdate () {
+	    if(UnityEngine.VR.VRDevice.isPresent)
+        {
+
+            Quaternion quatRot = UnityEngine.VR.InputTracking.GetLocalRotation(VRHead);
+            Vector3 vecPos = UnityEngine.VR.InputTracking.GetLocalPosition(VRHead);
+
+            // output to listener 
+
+            //float scale = 0.1f;
```

Comment:
Which numbers? You mean the whole equation thing? It's currently not implemented, because it slightly broken. Regardless, this particular file is in a realllly rough state. It hasn't been cleaned up and needs a fair amount of work at a later date to fix a ton of stuff.